apps/web/src/initializeApp.ts

```diff
@@ -57,8 +56,4 @@ export const initializeApp = () => {
       },
     });
   }
-
-  const tokenStoredToken: string = getToken();
-
-  applyToken(tokenStoredToken);
```

Comment:
We are no longer using the `defaults` config of Axios, so this is no longer required during initialization.